packages/core-api/src/apis/definitions/auth.ts

```diff
@@ -157,25 +157,47 @@ export type ProfileInfoOptions = {
   optional?: boolean;
 };
 
+/**
+ * This API provides access to profile information of the signed in user.
+ */
 export type ProfileInfoApi = {
   getProfile(options?: ProfileInfoOptions): Promise<ProfileInfo | undefined>;
 };
 
+/**
+ * Profile information of a signed in user.
```

Comment:
It's profile info for this auth provider, which might not be the same as the signed in user